sieve.py

```diff
@@ -548,16 +549,14 @@ def run(
                 phase,
             )
         else:
+            s = time.time()
             test_config = config
             print("Testing with config: %s" % test_config)
             test_config_to_use = os.path.join(log_dir, os.path.basename(test_config))
             cmd_early_exit("cp %s %s" % (test_config, test_config_to_use))
             if mode == sieve_modes.TIME_TRAVEL and suite.num_apiservers < 3:
                 suite.num_apiservers = 3
-            elif suite.use_csi_driver:
```

Comment:
@tylergu 
We need to keep `elif suite.use_csi_driver:` here. Can you add this back?

Reply:
nice catch, fixed it in e5ebee533245cc564aabc232ce77e2d2eecdf695